README.md

```diff
@@ -43,8 +43,7 @@ You can call this "protocol-all" if you like.  Unless, specified, always run com
     ```
     Note: if this is your first time you'll need to manually pull the latest bcgov image:
     ```
-    docker pull bcgovimages/aries-cloudagent:py36-1.15-1_0.6.0
-    ```
```

Comment:
I think removing this was probably an accident?

Reply:
Yep this was a mistake. For some reason VS code has been doing this in my searches